docs/content/en/schemas/v2beta7.json

```diff
@@ -1601,6 +1601,11 @@
     },
     "KptDeploy": {
       "properties": {
+        "applyDir": {
+          "type": "string",
+          "description": "path to the directory to deploy to the cluster.",
+          "x-intellij-html-description": "path to the directory to deploy to the cluster."
+        },
```

Comment:
Is that possible to use v2beta8.json (if not, I'm fine with adding the applyDir here).

It's safer to surface the kpt options after all the backend supports are done.

Reply:
The schemas are autogenerated. It is possible to use v2beta8 but that would require having latest config = v2beta7 and then created a "next latest" config = v2beta8. This seems a bit awkward.

Reply:
I see. Is v2beta7 the next new version or current version?

Reply:
v2beta7 hasn't been released yet, but is the version that is being developed and still allows for changes. Once we do another skaffold release, this version will become locked and someone will have to generate v2beta8

Reply:
v2beta7 is the upcoming version and v2beta6 is the current version.